emulator/src/main/java/com/scalar/client/tool/emulator/command/Database.java

```diff
@@ -0,0 +1,290 @@
+import com.scalar.database.io.TextValue;
+import com.scalar.database.io.Value;
+import com.scalar.ledger.emulator.MutableDatabaseEmulator;
+import java.io.StringReader;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+import java.util.Optional;
+import javax.inject.Inject;
+import javax.json.Json;
+import javax.json.JsonArrayBuilder;
+import javax.json.JsonNumber;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+import javax.json.JsonReader;
+import javax.json.JsonString;
+import javax.json.JsonValue;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+    name = "database",
+    sortOptions = false,
+    usageHelpWidth = TerminalWrapper.USAGE_HELP_WIDTH,
+    headerHeading = "%n@|bold,underline Usage|@:%n",
+    synopsisHeading = "",
+    descriptionHeading = "%n@|bold,underline Description|@:%n",
+    description = "Use this command to manipulate the mutable database built in the emulator",
+    parameterListHeading = "%n@|bold,underline Parameters|@:%n",
+    optionListHeading = "%n@|bold,underline Options|@:%n",
+    footerHeading = "%n",
+    footer = "Usage example: 'database'.%n")
+public class Database implements Runnable {
+  @CommandLine.Parameters(
+      index = "0",
+      paramLabel = "method",
+      description = "method that will be execute by MutableDatabase, e.g 'get'")
+  private String method;
+
+  @CommandLine.Option(
+      names = {"-p", "--primarykey"},
+      paramLabel = "key",
+      description = "the primary key (in JSON) that is to be inserted into the MutableDatabase")
+  private String primaryKey;
+
+  @CommandLine.Option(
+      names = {"-c", "--clusteringkey"},
+      paramLabel = "key",
+      description = "the primary key (in JSON) that is to be inserted into the MutableDatabase")
+  private String clusteringKey;
+
+  @CommandLine.Option(
+      names = {"-v", "--value"},
+      paramLabel = "value",
+      description = "the value (in JSON) that is to be inserted into the MutableDatabase")
+  private String value;
+
+  @CommandLine.Option(
+      names = {"-n", "--namespace"},
+      paramLabel = "namespace",
+      description = "the namespace of the database")
+  private String namespace;
+
+  @CommandLine.Option(
+      names = {"-t", "--table"},
+      paramLabel = "table",
+      description = "the table of the database")
+  private String table;
+
+  private MutableDatabaseEmulator databaseEmulator;
+
+  @Inject
+  public Database(MutableDatabaseEmulator databaseEmulator) {
+    this.databaseEmulator = databaseEmulator;
+  }
+
+  @Override
+  public void run() {
+    try {
+      switch (method) {
+        case "get":
+          runGet();
+          break;
+        case "delete":
+          runDelete();
+          break;
+        case "put":
+          runPut();
+          break;
+        case "scan":
+          runScan();
+          break;
+      }
+    } catch (IllegalArgumentException e) {
+      System.out.println(e.getMessage());
```

Comment:
Should it be 
```suggestion
      System.err.println(e.getMessage());
```
?

Reply:
I just realized we should display this message to `terminal` object.
This will dump string to CLI terminal.